bokeh/core/_templates/autoload_nb_js.js

```diff
@@ -138,7 +140,9 @@
   function display_loaded() {
     if (root.Bokeh !== undefined) {
       var el = document.getElementById({{ elementid|json }});
-      el.textContent = "BokehJS " + Bokeh.version + " successfully loaded.";
+      if (el !== undefined) {
```

Comment:
`getElementById()` returns `null` if element is not found. You should use `el != null` idiom here.